src/main/java/io/nats/client/impl/ErrorListenerConsoleImpl.java

```diff
@@ -23,31 +23,31 @@ public class ErrorListenerConsoleImpl implements ErrorListener {
      */
     @Override
     public void errorOccurred(final Connection conn, final String error) {
-        System.out.println(supplyMessage("[SEVERE] errorOccurred", conn, null, null, "Error: ", error));
+        log("[SEVERE]", "errorOccurred", conn, null, null, "Error: ", error);
```

Comment:
No. 
1. This is perfectly fine. A user can always make their own.
2. This would actually be a breaking change. Someone could rely out the specific text that this class outputs to a log and that would break their system.